### DISCUSSION TITLE
GitHub Feedback

### DISCUSSION THREAD

**Discussion by Sargeras02** [SKitLs-dev/SKitLs.Bots.Telegram#1] [Ideas]
## Welcome!
  It is my first published global project, so I literally have no idea, what I should to type in description, README or wiki.
  If you have any ideas, suggestions or questions, that could be mentioned in GitHub, don't be shy and share them. I'll be glad to see from you:
  * **Questions.** Questions about how works this or that.
  * **Suggestions.** If you faced any issues, understanding my code, so now you want to make this thing clear, you can also publish it here.
  * **Ideas.** In case you have any ideas of how make this project better (not in code, but in publishing), you also can be heard in this discussion.

**Comment by DrGrand03** [comment]
Hello.
I came across this project and decided to try to use it. Unfortunately, I encountered difficulties already at the quick start stage.(CORE)
If I understand correctly, then the description no longer corresponds to reality. For example, I don’t see classes such as DefaultSignedMessageUpdateHandler, DefaultSignedMessageTextUpdateHandler, DefaultActionManager.
If it is possible to correct the examples it would be great. Unfortunately, it’s not very convenient to run through the code looking for changes. 
Thank you

**Reply by SKitLs-dev** [reply]
Hi. Thanks for your feedback. I'm currently working on several projects solo.
I'm really sorry, it's quite hard to keep up with everything. I'm planning to update docs soon (1-2 weeks).
If you have any questions feel free to ask me.
